device.c

```diff
@@ -309,7 +309,11 @@ bool cSatipDevice::ProvidesChannel(const cChannel *channelP, int priorityP, bool
 
 bool cSatipDevice::ProvidesEIT(void) const
 {
+#if defined(APIVERSNUM) && APIVERSNUM < 20404
```

Comment:
I looked at VDR's git repo and API version 20403 supports this new device hook, so let's update this accordingly.

Reply:
Well, the reason why i choose 2.4.4 was, that there never was an release version with api 2.4.3; only dev on git.tvdr.de.

Otherwise, doesnt hurt. Done.